editor.planx.uk/src/@planx/components/AddressInput/AddressInput.stories.tsx

```diff
@@ -0,0 +1,55 @@
+import { Meta, StoryObj } from "@storybook/react";
+import { userEvent, within } from "@storybook/testing-library";
+import React from "react";
+
+import Wrapper from "../fixtures/Wrapper";
+import Editor from "./Editor";
+import Public from "./Public";
+
+export default {
+  title: "PlanX Components/AddressInput",
+  component: Public,
+} as Meta;
+
+export const EmptyForm: StoryObj = {
+  args: {
+    title: "Enter your address",
+    description: "It might be where you live",
+  },
+};
+
+export const EmptyFormWithHelpText: StoryObj = {
+  args: {
+    ...EmptyForm.args,
+    howMeasured: "This is an example definition.",
+  },
+};
+
+export const FilledForm: StoryObj = {
```

Comment:
This raises an interesting question about who Storybook is for / what do we want to get out of it (doesn't need an answer right away!).

I'd be inclined to see this more as a design example of error handling / error state, and then use the `play` API for more complex components that would maybe benefit from this visual demonstration (e.g. Upload and label, draw boundary). Appreciate this is just a demo of what v7 can do, but these are questions it would be good to cover on dev calls 👍

Reply:
On the `expect()` point - lets look into this properly 🕵️ 

If Storybook can give us a more robust environment for a11y and component level unit testing then that would be great.

Reply:
Yep - one for dev calls - I see this chat & decision around testing boundaries as it's own set of tasks ! Trying to keep focus to "a working, updated story file per component" here for starters.

Here's a nice example of `play` though that goes a couple steps further than I have: https://storybook.js.org/docs/react/api/csf#play-function (I've held off on lots of new dependencies for now, this would require at least `@storybook/jest` too)